content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+1. Grant permissions to the ServiceAccount object using an authorization
+   mechanism such as RBAC.
+1. Assign the ServiceAccount object to Pods during Pod creation.
+
+For instructions, refer to
+[Configure Service Accounts for Pods](/docs/tasks/configure-pod-container/configure-service-account/).
+
+### Grant permissions to a ServiceAccount
+
+You can use the built-in Kubernetes
+[role-based access control (RBAC)](/docs/reference/access-authn-authz/rbac/)
+mechanism to grant the minimum permissions required by each service account.
+You create a *role*, which grants access, and then *bind* the role to your
+ServiceAccount. RBAC lets you define a minimum set of permissions so that the
+service account permissions follow the principle of least privilege. Pods that
+use that service account don't get more permissions than are required to
+function correctly.
+
+#### Cross-namespace access using a ServiceAccount
+
+You can use RBAC to allow service accounts in one namespace to perform actions
+on resources in a different namespace in the cluster. For example, consider a
+scenario where you have a service account and Pod in the `dev` namespace and
+you want your Pod to see Jobs running in the `batch` namespace. You could
+create a Role object that grants permissions to list Job objects. Then,
+you'd create a RoleBinding object in the `batch` namespace to bind the Role
+to the ServiceAccount object. Now, Pods in the `dev` namespace can list Job
+objects in the `batch` namespace using that service account.
+
+### Use ServiceAccount credentials from a Pod
+
+To assign a service account to a Pod, you provide the Pod with the credentials
+for the service account, which the Pod can then use to authenticate with the
+Kubernetes API server. You can provide a Pod with credentials in the following
+ways:
+
+*   [TokenRequest API](/docs/reference/kubernetes-api/authentication-resources/token-request-v1/)
+    (recommended): Request a short-lived service account token using your
+    *application code*. The token expires automatically and can rotate upon
+    expiration.
+*   [Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
+    (recommended): In Kubernetes v1.20 and later, use the Pod specification to
+    tell the kubelet to add the service account token to the Pod as a
+    *projected volume*. Projected tokens expire automatically, and the kubelet
+    rotates the token before it expires.
+*   [Service Account Token Secrets](/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-service-account-api-token)
+    (not recommended): You can mount service account tokens as Kubernetes
+    Secrets in Pods. These tokens don't expire and don't rotate. This method
+    is not recommended, especially at scale, because of the risks associated
+    with static, long-lived credentials.
+
+## Authenticating service account credentials
```

Comment:
nit: I was confused by the title into thinking this was describing how to validate credentials, but that's the next section. Maybe a title tweak like this:

```suggestion
## Authenticating using service account credentials
```